subtitles.go

```diff
@@ -327,7 +339,9 @@ func (l Line) String() string {
 	for _, i := range l.Items {
 		texts = append(texts, i.Text)
 	}
-	return strings.Join(texts, " ")
+	t := strings.Join(texts, " ")
+	t = strings.ReplaceAll(t, "  ", " ")
```

Comment:
Also, can you provide an example `ttml` that was having issue with that bit of code?

Reply:
I used the same ttml in testdata i.e. example-in.ttml

While converting from TTML -> VTT 
TTML: 
` <p xml:id="sub_2" begin="00:02:04.080" end="00:02:07.120" region="region_2">
                MAN:
                <tt:br/>
                How did we <span style="style_1" color="green">end up</span> here?
            </p>`
 VTT:
Expected: How did we end up here?
Actual: How did weend uphere?